label/src/Label.js

```diff
@@ -22,12 +24,10 @@ export class Caption extends HTMLElement {
     const width = this.getAttribute('width');
     const widthStyle = width ? `style="width:${width};"` : '';
 
-    const text = this.getAttribute('text');
-
     this.shadowRoot.innerHTML = /*html*/ `
-      <style>${captionStyles}</style>
+      <style>${labelStyles}</style>
       <div class="wrapper">
-        <pre ${widthStyle} class="text">${text}</pre>
+        <pre ${widthStyle} class="text"><slot></slot></pre>
```

Comment:
AFAICS this is the main change, isn't it? why do you need a slot?

Reply:
I think it's more natural to have the text as a child:
`<dockit-label>some text</dockit-label>`
vs.
`<dockit-label text="some text"></dockit-label>`

Disclaimer: I might be wrong about how slots work :D

Reply:
typical reason to use a slot is to allow HTML natively, because otherwise it's not going to be rendered withouts hacks like innerHTML

we use innerHTML right now, so even `text` prop would work, but only because we do not sanitize the input which we should do at some point to improve on the security part

so let's assume security is improvement, then only slot will allow you to pass HTML to this component
if this is your goal, then I'm OK, otherwise prop is better, because it communicates well that only pure text is allowed which is also a good way to design the API